src/actions/utils/ActionData.sol

```diff
@@ -0,0 +1,15 @@
+/**
+ * Created by Arcadia Finance
+ * https://www.arcadia.finance
+ *
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity >=0.8.0 <0.9.0;
+
+struct actionAssetsData {
```

Comment:
Think the convention (at least what we did so far is to use capital letter here).
Would anyway remove the Assets
```suggestion
struct ActionData {
```

Reply:
it's not only about the action data though, also about which assets & the amounts

Reply:
maybe another name alltogether

Reply:
But that is all related to the action no?
Think ActionData is clear enough